src/acq.h

```diff
@@ -23,10 +23,23 @@
 
 union bl_msg_data;
 
-/** Acquisition mode */
-enum bl_acq_mode {
-	BL_ACQ_MODE_CONTINUOUS,
-	BL_ACQ_MODE_FLASH,
+/** SPI mode**/
+extern enum bl_acq_spi_mode {
+	BL_ACQ_SPI_NONE,
+	BL_ACQ_SPI_MOTHER,
+	BL_ACQ_SPI_DAUGHTER,
+} bl_spi_mode;
```

Comment:
```suggestion
};
```

You can't declare this in the header or every compilation unit will have its own separate copy, which *might* explain why it sometimes works and sometimes not.

I'd put something like this:

```c
enum bl_acq_spi_mode bl_spi_mode;
```

in `src/spi.c`.

Reply:
Oh, I found it in `src/bl.c`!  Never mind.